src/changelog/getPackageChangelogs.ts

```diff
@@ -22,7 +22,7 @@ export function getPackageChangelogs(
   const commit = getCurrentHash(cwd) || 'not available';
 
   for (let change of changeInfos) {
```

Comment:
I think the only place that this dependentChange flag is used is here. We already know which changeInfo is dependentChange because of the concat above. We could just break apart the loop here into two (changeFileChangeInfos + dependentChangeInfos). Then only execute the 2nd loop if the new "ignore flag" is true.

Further, I'm not convinced I would ever include the dependentChangeInfo for grouped messages, so maybe we just don't even pass that in at all from the writeGroupedChangelog() in the first place?

Reply:
That's a good callout, it makes the whole thing a lot easier, and the tests still pass as expected 👍

Reply:
I'm not convinced we need changes to the "dependentChange" tracking at all anymore... so this isn't needed?

Reply:
Yup removed, failed to check in the commit 😅